File: app/src/main/java/com/shevart/androidcorelearn/utils/UiUtil.java
package com.shevart.androidcorelearn.utils;

import android.support.annotation.LayoutRes;
import android.support.annotation.NonNull;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class UiUtil {
    public static View inflate(@NonNull ViewGroup parent, @LayoutRes int layoutRes) {
        return LayoutInflater.from(parent.getContext()).inflate(layoutRes, parent, false);
    }
}

File: app/src/main/java/com/shevart/androidcorelearn/utils/UiUtil.kt
package com.shevart.androidcorelearn.utils

import android.support.annotation.LayoutRes
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

object UiUtil {
    fun inflate(parent: ViewGroup, @LayoutRes layoutRes: Int): View {
        return LayoutInflater.from(parent.context).inflate(layoutRes, parent, false)
    }
}
